### ISSUE TITLE
Question about the learning rate used in linear evaluation

### ISSUE THREAD

**Issue by Vickeyhw** [facebookresearch/simsiam#13]
It seems that the learning rate mentioned in paper is different from the code . The paper said'lr=0.02 for 90 epochs with batch size=4096 with a LARS optimizer', but  in code, according to the default setup , lr=0.1 is used for batch size 256, so for batch size 4096, learning rate is scaled up use 'init_lr = args.lr * args.batch_size / 256' .   So how should I set learning rate in linear evaluation?

**Comment by endernewton** [comment]
The paper described the setting we used on the TPU side with TF. The code base is implemented in GPU w/ PyTorch which I find 0.1 works quite well and always stick to it (we have not tried 0.02 with this code but I believe it should also achieve reasonable results). The cause is unknown, maybe some subtle difference between GPU/TPU/TF/PT.
